stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -1152,7 +1163,7 @@ extension Unsafe${Mutable}RawBufferPointer {
 %  end
   @inlinable
   @_alwaysEmitIntoClient
-  public func withContiguousStorageIfAvailable<R>(
+  public func withContiguousStorageIfAvailable<R: ~Copyable>(
```

Comment:
Should we add this on `withContiguousStorageIfAvailable()` if we haven't done so for `Sequence`?